include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp

```diff
@@ -177,6 +186,22 @@ class rofl_ofdpa_fm_driver final {
                                       const caddress_in6 &dst,
                                       const caddress_in6 &mask,
                                       uint16_t vrf_id = 0);
+  // Multicast Routing
+  cofgroupmod enable_group_l3_multicast(uint8_t ofp_version, uint32_t port_id,
```

Comment:
the disable function is missing here (not that I like adding functions that are not used but if you really want to add them please be consistent)